budgets/tests.py

```diff
@@ -11,10 +11,10 @@
 
 class BudgetsTests(TestCase):
     def setUp(self):
-        userModel = get_user_model()
-        self.user = userModel(username='test')
+        user_model = get_user_model()
```

Comment:
Here is better to rename userModel to UserModel as it is class not an ordinary variable